examples.py

```diff
@@ -0,0 +1,55 @@
+# %%
```

Comment:
Please add these code snippets to Readme
Create a separate directory called examples and write these examples in separate files.